pkgs/servers/elasticmq-server-bin/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, jdk, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "elasticmq-server-${version}";
```

```suggestion
  pname = "elasticmq-server";
```
 the default value of name is `name = "${pname}-${version}"